src/Umbraco.Infrastructure/Migrations/Install/DatabaseDataCreator.cs

```diff
@@ -539,7 +539,7 @@ void InsertDataTypeNodeDto(int id, int sortOrder, string uniqueId, string text)
                 Path = $"-1,{Constants.DataTypes.DateTime}",
                 SortOrder = 2,
                 UniqueId = Constants.DataTypes.Guids.DatePickerWithTimeGuid,
-                Text = "Date Picker with time",
+                Text = "Date Picker With Time",
```

Comment:
English language style guides disagree as to whether four-letter propositions should be capitalised in title case. So either "Date Picker with Time" or "Date Picker With Time" could be valid here.

It's time to be opinionated. I prefer the former, which anecdotally seems more common. Other opinions welcome.

```suggestion
                Text = "Date Picker with Time",
```

Reply:
Tagging @nathanwoulfe, thanks for making this easier 😅
```suggestion
                Text = "Date and Time Picker",
```